core/tests/test_views.py

```diff
@@ -37,27 +40,46 @@ def community() -> Community:
 
 
 def test_add_post_valid(client: Client, user: User, community: Community) -> None:
-    data = {
-        "community": community.pk,
-        "title": "Test Post Title",
-        "content": "This is a test post content.",
-    }
-    client.force_login(user)
-    response = client.post(reverse("post-create"), data=data, follow=True)
-    assert response.status_code == 200
-    assert response.context["post"].author == user
-    assert response.context["post"].title == data["title"]
-    assert response.context["post"].content == data["content"]
+    with tempfile.NamedTemporaryFile(suffix=".jpg", delete=False) as tmp_file:
```

Comment:
The delete parameter should be true, so the file is deleted when the test is finished, no? We don't want to have mess on the drive.

Reply:
You're right